docs/core-concepts/model-basics.md

```diff
@@ -278,6 +278,35 @@ Foo.init({ /* attributes */ }, {
 });
 ```
 
+## Prevent creating a default PK attribute
+
+By default, Sequelize automatically adds the primary key field `id` to every model when there is no primary key field defined and would throw an error when there is an `id` field but not a primary key field, to prevent this you can use the option `noPrimaryKey` and set it to true.
+
+```js
+const sequelize = Sequelize.createSequelizeInstance({
+  define: {
+    noPrimaryKey: true,
+  },
+});
+
+const User = sequelize.define('User', {
+  name: {
+    type: DataTypes.STRING
+  }
+});
+```
+
+In case you do not want to use `noPrimaryKey` and do not want to caught by an error you can define `primaryKey` on the model fields and set it to false.
```

Comment:
Maybe rephrase this so it's more clear that this will then happen only on a single model.

Something like: 'In case you only want to disable primary keys on a single model, you can define the field with `primaryKey: false`'

Reply:
Thanks